### PR TITLE
Fixes plant analyzer and hydroponics tray information

### PR DESCRIPTION
**Pull request by Ithalan**
Fixes plant analyzer not showing data about gasses consumed or exuded by plants
Fixes hydroponics trays not showing sensor information when empty

### CODE REVIEW

File: code/modules/hydroponics/trays/tray_tools.dm [hunk 1]
```diff
@@ -197,18 +201,24 @@
...
 	if(grown_seed.get_trait(TRAIT_TELEPORTING))
 		dat += "<br>The fruit is temporal/spatially unstable."
-	
-	if(grown_seed.get_trait(TRAIT_EXUDE_GASSES))
-		dat += "<br>It will release gas into the environment."
-
-	if(grown_seed.get_trait(TRAIT_CONSUME_GASSES))
-		dat += "<br>It will remove gas from the environment."
-
-	if(grown_seed.get_trait(TRAIT_EXUDE_GASSES))
```
Comment (Zuhayr) [inline code/modules/hydroponics/trays/tray_tools.dm]: I wonder why this was done twice?

Comment (Datraen) [inline code/modules/hydroponics/trays/tray_tools.dm]: Because a commit didn't make it into the PR I did fixing that, I guess.

**Comment by Datraen** [comment]
It was purposely excluded for Xenobotanists, who have access to gas scanners, to be able to experimentally figure out what the gases are, as a way to help discourage gardeners ordering exotic seeds.

**Comment by Ithalan** [comment]
Couldn't that be accomplished much more effectively by just making the exotic seeds crate require xenoflora access to unlock?

Every other plant property can be identified either through the analyzer or visual inspection of the plant icon, so hiding the existence of the gas property specifically just seems like a pointless inconsistency waiting to fuck new players over (I certainly didn't find it enjoyable to be screamed at by others for creating overpressure the first time I planted some exotic seeds, when there was no way outside of codediving that I could have known that the gas property could exist.)

**Comment by Datraen** [comment]
The plant analyzer was showing whether or not it exuded or consumed gas, if you scanned the seeds prior to planting them, you would see it. 

Yes, we could lock the crates to xenoflora access, but at the time that it was done, plants were very much the same, and it was the only variable that wasn't directly given to you by the scanner, save for the contents of the fruit.

**Comment by Datraen** [comment]
For reference: #925 was a related bug report

**Pull request merged**